static/app/views/discover/tags.spec.tsx

```diff
@@ -7,8 +7,27 @@ import {
 } from 'sentry-test/reactTestingLibrary';
 
 import EventView from 'sentry/utils/discover/eventView';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
 import {Tags} from 'sentry/views/discover/tags';
 
+// There seem to be no types for this, but it's essentially what is passed to the
+// EventView constructor
+const commonQueryConditions = {
+  additionalConditions: new MutableSearch([]),
+  display: '',
+  start: '',
+  end: '',
+  id: '',
+  name: '',
+  project: [1],
+  environment: [],
+  topEvents: '',
+  yAxis: '',
+  createdBy: TestStubs.User(),
+  team: TestStubs.Team(),
+  statsPeriod: '14d',
+};
+
```

Comment:
Ack, I just ran [into the same thing in my own PR](https://github.com/getsentry/sentry/pull/52288/files#diff-3fdda4e13244598e290f5864381e2490abf5467fb8a4dec889be28ac547d0d88R32-R48)

I ended up doing `ConstructorParameters<typeof EventView>[0]` which is icky but works